### DISCUSSION TITLE
Why does the If activity in Elsa 3.1.3 designer only have the Condition attribute and not the Then and Else attributes?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5678] [General]
Elsa 3.1.3 designer seems to have an issue where Then and Else properties are not visible in the designer interface.
Why does the If activity in Elsa 3.1.3 designer only have the Condition attribute and not the Then and Else attributes?
![If](https://github.com/elsa-workflows/elsa-core/assets/171897414/d780fe21-9377-4dbb-b6d4-cac915f00950)

**Comment by glime-ai** [comment]
Based on the context provided, it appears that the issue with the If activity in Elsa 3.1.3 designer not showing the Then and Else attributes is a known problem. This could be due to a limitation or a bug in the current version of the Elsa Workflow designer. The Elsa Workflow GitHub issues and discussions do not provide a direct answer to this specific question, but similar issues have been reported regarding the support for certain activities in the designer interface.

It is recommended to check the Elsa GitHub repository for any open issues related to the If activity or to report this as a new issue if it has not been documented yet. Additionally, you can reach out to the Elsa community on their Discord channel or Stack Overflow for further assistance and to see if there are any workarounds or updates on this matter.

Would you like me to provide guidance on how to report an issue on GitHub or seek help from the community?

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
